binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,120 @@
+class Node(object):
+    # Node for Binary search Tree and set the Node
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+# insert data in tree
+def insert(node, data):
+    if node is None:
+        # create a new node
+        node = Node(data)
+        return node
+    else:
+        # go to the left child
+        if data <= node.data:
+            node.left = insert(node.left, data)
+        # got to the right child
+        else:
+            node.right = insert(node.right, data)
+    return node
+
+# seraching data
+def search(node, data):
+    # if data not present
+    if node is None:
+        return None
+    # go to the left if data is lesser from root
+    if data < node.data:
+        node = search(node.left, data)
+    # go to the right if data is greater from root
+    elif data > node.data:
+        node = search(node.right, data)
+    # data found
+    elif data == node.data:
+        return node
+        return node
+
+# find the minimum
+def minright(node):
+    if node.left is None:
+        return node
+    else:
+        node = minright(node.left)
+    return node
+
+# delete a node
+def delete(root, data):
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmppakk8uf0/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmppakk8uf0/binary_search_tree/BinarySearchTree.py
@@ -21,6 +21,8 @@
     return node
 
 # seraching data
+
+
 def search(node, data):
     # if data not present
     if node is None:
```